README.md

```diff
@@ -45,13 +45,14 @@ To test the bot, you can:
 
 1. Create a new issue
 2. Add a time label, ex: `Time: <1 Day`
-3. At this point the bot should add a price label.
+3. Add a priority label, ex: `Priority: 0 (Normal)`
+4. At this point the bot should add a price label.
 
 ## Configuration
 
-`chain-id` is ID of the EVM-compatible network that will be used for payouts.
+`evm-network-id` is ID of the EVM-compatible network that will be used for payouts.
 
-`base-multiplier` is a base number that will be used to calculate bounty price based on the following formula: `price = base-multiplier * time-label-weight * priority-label-weight / 10`
+`price-multiplier` is a base number that will be used to calculate bounty price based on the following formula: `price = price-multiplier * time-label-weight * priority-label-weight / 10`
```

Comment:
Do we still have `/ 10`?

Reply:
It appears it is 100 now.

Even though looks like 1000.
https://github.com/ubiquity/ubiquibot/blob/d56a523c81829e17513f7da1f1ccb1bbdca8b180/src/handlers/shared/pricing.ts#L7

but priority is being being divided by 10 too. 
https://github.com/ubiquity/ubiquibot/blob/d56a523c81829e17513f7da1f1ccb1bbdca8b180/src/handlers/shared/pricing.ts#L6